tests/integration/api/controllers/all-docs.spec.js

```diff
@@ -2,6 +2,8 @@ const chai = require('chai');
 const _ = require('lodash');
 const utils = require('../../../utils');
 const constants = require('../../../constants');
+const chaiExclude = require('chai-exclude');
```

Comment:
I don't see `exclude` feature being used anywhere, do we still want to introduce it?

Reply:
@meghna-khemka  I think that's the `excludingEvery` used in a few places below.